### PR TITLE
Danielle baxter

### PR DESCRIPTION
**Pull request by danibaxx**
Submitting PR for PM review. Thanks!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,16 +4,78 @@
...
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
 
 <body>
-  <h1>My Resume</h1>
+  <div class="container">
+    <header>
+      <nav>
+        <a href="https://danibaxx.github.io/portfolio-website/">Home</a>
```
Comment (ari-mercado) [inline index.html:24]: Perhaps the home can link to the Resume and the Portfolio anchor tag link to your Portfolio?

File: less/index.less [hunk 2]
```diff
@@ -54,6 +54,249 @@ table {
...
+	background: mediumslateblue;
+	color: whitesmoke;
+}
+
+
+// Nesting top container
+
+.top-container {
+	.container-style;
+	background-image: linear-gradient(darkslateblue, darkslateblue, whitesmoke);
```
Comment (ari-mercado) [inline less/index.less:183]: This linear-gradient looks very nice!

File: less/index.less [hunk 3]
```diff
@@ -54,6 +54,249 @@ table {
...
+		width: 100%;
+	}
+}
+
+ul {
+	list-style-type: square;
+	padding: 0 25px;
+	line-height: 2.1;
+	font-size: 1.3rem;
+	border: 3px dashed midnightblue;
```
Comment (ari-mercado) [inline less/index.less:259]: Considering you listed "working with multiple opinions" as something your good at, in my opinion, this dashed border is not needed to visually separate your description and the unordered list. Also, the bullet points might also not be needed. Often times, this visual separation can look cleaner and work better with enough whitespace.
https://blog.teamtreehouse.com/white-space-in-web-design-what-it-is-and-why-you-should-use-it.

**Comment by ari7946** [comment]
Your digital resume looks great, Danielle! The code is very clean and easy to read. I like the number of descriptive commit messages. Keep up the great work!

**Pull request merged**